test/unit/messaging/process-return-message.test.js

```diff
@@ -1,25 +1,91 @@
 jest.mock('ffc-messaging')
+
 const processReturnMessage = require('../../../app/messaging/process-return-message')
+
+jest.mock('../../../app/inbound/process-return-settlement')
+const processReturnSettlement = require('../../../app/inbound/process-return-settlement')
+
 let receiver
 let settlement
+let message
 
 describe('process return message', () => {
   beforeEach(() => {
+    processReturnSettlement.mockReturnValue(undefined)
     settlement = JSON.parse(JSON.stringify(require('../../mock-settlement')))
     receiver = {
       completeMessage: jest.fn()
     }
+    message = { body: { settlement } }
```

Comment:
would also change this (tht includes you evil _Marc_) to `message = { body: settlement }`